src/main/webapp/css/style-color-blind.css

```diff
@@ -0,0 +1,123 @@
+table.source {
```

Comment:
My concern with duplicating the entire css file is that anyone wanting to change the layout would have to change in two places (and might miss this one). Looking at the layout control https://github.com/jenkinsci/jenkins/blob/master/core/src/main/resources/lib/layout/layout.jelly#L46-L49 I see the `css` parameter is deprecated. 

This suggests a slightly different approach, which would be to move the css into a `<script>` tag inside of an `<l:style>` control. Then we could provide a color-blind css file that has just the overrides (presumably the colors). For regular mode, just import the base stylesheet, and for color-blind mode import the base stylesheet, then import the color-blind stylesheet. This has the advantage that we could easily provide more than one color-blind mode in the future should we want to.

Reply:
Oh, You are right. Thanks for your advice :)